views/project/_card.php

```diff
@@ -9,8 +9,17 @@
 
 ?>
 
-<a class="project-card <?= $model->getStatusClass() ?>" href="<?= Url::to(['project/view', 'id' => $model->id, 'slug' => $model->slug]) ?>">
-    <img class="img-responsive" src="<?= $model->getPrimaryImageThumbnailRelativeUrl() ?>" alt="">
-    <h1><?= Html::encode($model->title) ?></h1>
+<article class="<?= $model->getStatusClass() ?>">
+    <a class="details" href="<?= Url::to(['project/view', 'id' => $model->id, 'slug' => $model->slug]) ?>">
+        <img class="img-responsive" src="<?= $model->getPrimaryImageThumbnailRelativeUrl() ?>" />
+        <h1><?= Html::encode($model->title) ?></h1>
+    </a>
+
+    <p class="tags">
+        <?= implode(', ', array_map(function ($tag) {
+            return Html::a(Html::encode($tag->name), ['/project/list', 'tags' => $tag->name]);
+        }, array_slice($model->tags, 0, 10))); ?>
```

Comment:
It causes lots and lots of SQL queries. `with()` should be used when selecting data.